### PR TITLE
FX2024 Domain & Capabilities update / Understand cloud usage and cost

### PR DESCRIPTION
**Pull request by arthurclares**
## 🛠️ Description
This PR updates all the capabilities that are part of the understand cloud usage and cost  domain according to the new FinOps framework.

### 🔬 How did you test this change?

> - [ ] 🤏 Lint tests
> - [ ] 🤞 PS -WhatIf / az validate
> - [ ] 👍 Manually deployed + verified
> - [ ] 💪 Unit tests
> - [ ] 🙌 Integration tests

### 🙋‍♀️ Do any of the following that apply?

> - [ ] 🚨 This is a breaking change.
> - [ ] 🤏 The change is less than 20 lines of code.

### 📑 Did you update `docs/changelog.md`?

> - [ ] ✅ Yes (required for `dev` PRs)
> - [x] ➡️ Will cover in a future PR (feature branch PRs only)
> - [ ] ❎ Not needed (small/internal change)

### 📖 Did you update documentation?

> - [x] ✅ Public docs in `docs` (required for `dev`)
> - [ ] ✅ Internal dev docs in `src` (required for `dev`)
> - [ ] ➡️ Will cover in a future PR (feature branch PRs only)
> - [ ] ❎ Not needed (small/internal change)

### CODE REVIEW

File: docs/_docs/framework/capabilities/understand/reporting.md
Comment (flanakin) [inline docs/_docs/framework/capabilities/understand/reporting.md:90]:
```suggestion
If there is the need to report from multiple sources, for example multiple cloud providers, configuration management database (CMDB), or  IT asset management (ITAM) systems, consider using the FinOps Open Cost & Usage Specification ([FOCUS project](../../../focus/README.md)) as the standard billing schema for this report.
```

File: docs/_docs/framework/capabilities/understand/reporting.md
Comment (flanakin) [inline docs/_docs/framework/capabilities/understand/reporting.md:123]:
```suggestion
- If you need to build more advanced reports or merge cost data with other cloud or business data, [leverage the FinOps toolkit Power BI reports](../../../../_reporting/power-bi/README.md) part of the open source FinOps toolkit provided by Microsoft.
```

File: docs/_docs/framework/capabilities/understand/reporting.md
Comment (flanakin) [inline docs/_docs/framework/capabilities/understand/reporting.md:125]:
```suggestion
    - FinOps hubs includes a [Power BI report](../../../../reporting/power-bi) that normalizes data to the FOCUS schema, which can be a good starting point.
```

File: docs/_docs/framework/capabilities/understand/reporting.md
Comment (flanakin) [inline docs/_docs/framework/capabilities/understand/reporting.md:154]:
```suggestion
This capability is a part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [Reporting and analytics capability](https://www.finops.org/framework/capabilities/reporting-analytics/) article in the FinOps Framework documentation.
```

File: docs/_docs/framework/capabilities/understand/reporting.md
Comment (flanakin) [inline docs/_docs/framework/capabilities/understand/reporting.md:159]:
```suggestion
{% include video.html title="Reporting and analytics videos" id="CVTJLdcozj1eEpxT" list="PLUSCToibAswlDSQdehKhi7ysP2hmetigl" %}
```

**Review by flanakin — CHANGES_REQUESTED** [review]

**Comment by arthurclares** [comment]
@flanakin All the changes are implemented. It's waiting on your approval now.

**Review by flanakin — APPROVED** [review]

**Pull request merged**
